src/app/account-transfers/edit-standing-instructions/edit-standing-instructions.component.ts

```diff
@@ -39,7 +39,7 @@ export class EditStandingInstructionsComponent implements OnInit {
   /** Minimum date allowed. */
   minDate = new Date(2000, 0, 1);
   /** Maximum date allowed. */
-  maxDate = new Date();
+  maxDate = new Date(2100,0,1);
```

Comment:
Do we need max date?

Reply:
Actually I don't know if we need or not, it was already in code and it was till business date and while creating standing instruction it was till this so I thought I should make it till this

Reply:
@adamsaghy should I remove max-date, if don't require?

Reply:
@bharathcgowda Do we need max date for standing instructions?

Reply:
@adamsaghy @PC-11-00  is this max-date same as the "valid to date" in the UI? then Yes it is required